doc/classes/Environment.xml

```diff
@@ -9,6 +9,7 @@
 		- Glow
 		- Tonemap (Auto Exposure)
 		- Adjustments
+		These effects will only apply when the [b]Intended Usage[/b] in the [b]Project Settings[/b] (under [b]General > Rendering > Quality[/b]) is set to [b]3D[/b] or [b]3D Without Effects[/b].
```

Comment:
As commented by @clayjohn, using hyperlinks would be more useful than a lot of bold text:

> you can update the sentence to hyperlink to the relevent sections:
"These effects will only apply when the [Viewport] [member usage] is [constant USAGE_3D] or [constant USAGE_3D_NO_EFFECTS]."

It's not 100% valid though as it reference `Viewport` properties and constants, it should be `[member Viewport.usage]`, `[constant Viewport.USAGE]`. And a reference to the project setting is indeed useful. I'd suggest:

"These effects will only apply when the [Viewport]'s intended usage is "3D" or "3D Without Effects". This can be configured for the root Viewport with [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation], or for specific Viewports via the [member Viewport.usage] property."

CC @clayjohn to review, I'm not sure how the project setting and the Viewport property interact with each other exactly.

Reply:
The project setting sets the values for the root Viewport. So both the project setting and the Viewport settings should be mentioned.